tools/slurm_train.sh

```diff
@@ -6,9 +6,9 @@ PARTITION=$1
 JOB_NAME=$2
 CONFIG=$3
 WORK_DIR=$4
-GPUS=${GPUS:-8}
+GPUS=${GPUS:-32}
 GPUS_PER_NODE=${GPUS_PER_NODE:-8}
-CPUS_PER_TASK=${CPUS_PER_TASK:-5}
+CPUS_PER_TASK=${CPUS_PER_TASK:-1}
```

Comment:
I think it is no need to change these two default value

Reply:
Thank you for your valuable reviews. For the convenience of debugging, I change the default value of `CPUS_PER_TASK` and forgot to change it back in my official PR. I will fix it~